lib/types/entry.ts

```diff
@@ -243,9 +250,11 @@ export type ResolvedLink<
     ? ResolvedEntryLink<Modifiers, Locales, LinkedEntry>
     : Field extends EntryFieldTypes.EntryResourceLink<infer LinkedEntry>
       ? ResolvedEntryResourceLink<Modifiers, Locales, LinkedEntry>
-      : Field extends EntryFieldTypes.AssetLink
-        ? ResolvedAssetLink<Modifiers, Locales>
-        : BaseFieldMap<Field>
+      : Field extends EntryFieldTypes.ExternalResourceLink
+        ? { sys: ResourceLink<string> }
+        : Field extends EntryFieldTypes.AssetLink
+          ? ResolvedAssetLink<Modifiers, Locales>
+          : BaseFieldMap<Field>
```

Comment:
is there no other way to represent this? 😬

Reply:
What do you mean by represent?